UACloudLibraryServer/Startup.cs

```diff
@@ -150,6 +150,7 @@ public void ConfigureServices(IServiceCollection services)
                     services.AddSingleton<IFileStorage, AWSFileStorage>();
                     break;
                 case "GCP": services.AddSingleton<IFileStorage, GCPFileStorage>(); break;
+                case "DevDB": services.AddScoped<IFileStorage, DevDbFileStorage>(); break;
```

Comment:
This is a great addition to testing scenarios, BTW...

Reply:
Yep, spent quite some time tracking down weird behavior when the stores where just out of sync ;-)